helpdesk_mgmt_fieldservice_equipment/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```

In 18.0, use `self.env._` instead